src/app/routes/enrolment/view-requests/issuer-requests/issuer-requests.component.ts

```diff
@@ -19,7 +19,7 @@ export class IssuerRequestsComponent
   extends ViewRequestsComponent
   implements OnInit
 {
-  @ViewChild('issuerFields', { static: false }) requiredFields: EnrolmentForm;
+  @ViewChild('issuerFields', { static: true }) requiredFields: EnrolmentForm;
```

Comment:
Was this change part of the fix? I'm reading this blog post to understand this better
https://angular.io/guide/static-query-migration

So is this change necessary because the query results need to be available in `ngOnInit`?

Reply:
It solved issue with `ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked`. If i enable prod build for dev env, then it will disappear in all places. However it solved issue here for non prod. `issuerFields` are not under `ngIf` so they can be static. If this would be parametrised, so issuerFields are toggled via ngIf, then it should be `static: false`.